crates/build/re_types_builder/src/codegen/rust/api.rs

```diff
@@ -1047,6 +1047,23 @@ fn quote_trait_impls_for_archetype(obj: &Object) -> TokenStream {
 
     assert_eq!(kind, &ObjectKind::Archetype);
 
+    // sanity check that each field has one of the rerun component type attributes
+    for field in &obj.fields {
+        assert!(
```

Comment:
can we do `reporter.error` with all the right paths & stuff please? That way it's way easier to find & address the error

Reply:
good call. done.